packages/antd/src/hooks/list/useSimpleList/useSimpleList.ts

```diff
@@ -35,6 +36,13 @@ export type useSimpleListReturnType<
     searchFormProps: FormProps<TSearchVariables>;
     filters: CrudFilters;
     setFilters: useTableReturnType<TData>["setFilters"];
+    sorter?: CrudSorting;
+    setSorter: useTableReturnType<TData>["setSorter"];
```

Comment:
```suggestion
    sorters?: CrudSorting;
    setSorters: useTableReturnType<TData>["setSorter"];
```
like `filters` and `setFilters`?